vos/vos/tests/test_vofile.py

```diff
@@ -1,4 +1,4 @@
-from builtins import object
+from future.builtins import object
```

Comment:
astropy recommends the six library as a way to link Python 2 and 3 and this is what I was trying to always use. future, however, seems to be more appropriate if 2.7 is the earliest supported version. Should we start using future then? I would say either one or the other. Do you have more experience with future?